server-plugins/love-resources/src/index.ts

```diff
@@ -313,7 +321,8 @@ export async function OnInvite (tx: Tx, control: TriggerControl): Promise<Tx[]>
               name: formatName(from.name, control.branding?.lastNameFirst)
             })
             : await translate(love.string.InivitingLabel, {})
-        await createPushNotification(control, userAcc._id, title, body, invite._id, from, path)
+        // TODO: Select a proper user
+        await createPushNotification(control, userAcc[0]._id, title, body, invite._id, from, path)
```

Comment:
In the current implementation of accounts, shouldn't we send notification to all user accounts?

Reply:
Let's let it be this way as current implementation, since we will rework it in near future.